classlib/src/main/java/org/teavm/classlib/java/util/TTreeMap.java

```diff
@@ -561,16 +563,16 @@ public Object clone() {
     }
 
     static class EntrySet<K, V> extends TAbstractSet<Entry<K, V>> implements TSequencedSet<Entry<K, V>> {
+        private final TTreeMap<K, V> owner;
```

Comment:
Changes like these give nothing, but make review harder

Reply:
Yes, it make review harder, but ordered firstly final and then not final fields improve readability significantly. Actually this readability helped alot to identify issues during fixing.

Reply:
For me not. For instance, in MapView modCount and cachedSize were on the top, entrySetCache and cachedNavigableSet were on the bottom. It turned out that modCount and cachedize are not needed. Still, this is mess in my opinion. Also there were some wrong-order passing of variables, it was hard to guess about it without looking at properly ordered final fields. That is the reason for ordering them and distinguishing non-final from final.

Reply:
Please, revert these changes. It improves readability only from your standpoint. Another reason: I'd like to refactor TreeNode to contain link to parent and these fields will be changed. So these changes are nothing more than git history pollution